bxml/bxmlConcepts.md

```diff
@@ -0,0 +1,64 @@
+# BXML Concepts and Vocabulary
+
+## XML
+
+XML is a metalanguage that allows a developer to build up their own customized coding language to perform certain tasks.
+
+[Click here to learn more](https://en.wikipedia.org/wiki/XML)
+
+## BXML
+
+Bandwidth XML (or BXML) is the markup language that Bandwidth developed to handle inbound and outbound call flows from within your application. BXML handles call events using standard action verbs that are listed in the BXML library.
+
+[Click here to learn more](bxml.md)
+
+## BXML Verbs
+
+The word that instigates the method. This is always first word and followed by any parameters such as a timeout parameter or the speak sentence voice parameter.
+
+[Click here to learn more](bxml.md)
+
+## REST API
+
+A REST API defines a set of functions where developers can perform requests and receive responses via an HTTP protocol such as GET and POST. Here at Bandwidth, we have a Voice and Messaging API that gives a user the capability to customizably send and receive calls or messages over the internet. We will use the REST API in BXML to make an outbound call.
+
+[Click here to learn more](methods/restApi.md)
```

Comment:
Link doesn't work... try `../methods/restApi.md` The paths to internal links are relative to where _this_ file is located.